quick_test.py

```diff
@@ -32,7 +32,7 @@
 
     else:
         tabpfn_classifier.init()
-        tabpfn = TabPFNClassifier()
+        tabpfn = TabPFNClassifier(model="latest_tabpfn_hosted")
         # print("checking estimator", check_estimator(tabpfn))
         print(X_train.shape[0]*100)
         tabpfn.fit(np.repeat(X_train, 100, axis=0), np.repeat(y_train, 100, axis=0))
```

Comment:
Since there was a bug here that we did not spot from the last PR, could you please add a small test that this file runs to the tests

Reply:
I get an error when I try to run this file:
RuntimeError: Fail to call upload_train_set with error: {'detail': 'The training dataset provided is too large, as the number of examples/lines is 38099 which is greater than the maximum number of lines (9999).'}
Is this intended?

Reply:
i might have added this during debugging. please change this s.t. it runs through, i.e. decrease number of examples to something super small like 100

Reply:
ah I sea, then the test is unnecessary!